### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by alexanderteplov**
Skipped:
 - Webpack
 - async/await
, as I don't understand why they are needed here. Think they're unnecessary.
Other points, including additional, were implemented.

### CODE REVIEW

File: product-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      bundle: true,
+      minify: true,
+      sourcemap: true,
+      exclude: ['aws-sdk'],
+      target: 'node14',
+      define: { 'require.resolve': undefined },
+      platform: 'node',
+      concurrency: 10,
+    },
+    documentation: {
```
Comment (maxm-dev) [inline product-service/serverless.ts:40]: [Minor]
It's better to move this section out of the global `serverless.ts` file to a separate module.

File: product-service/src/functions/getProductById/__tests__/getProductById.test.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    expect(body).toBe(undefined);
+  });
+
+  test('Should return 404 with incorrect productId', () => {
+    const incorrectPathParameters = { productId: '7' };
+    const { statusCode, body } = getProductById({ productList, pathParameters: incorrectPathParameters });
+    expect(statusCode).toBe(404);
+    expect(body).toBe(undefined);
+  });
+});
```
Comment (maxm-dev) [inline product-service/src/functions/getProductById/__tests__/getProductById.test.ts:38]: There is a good practice to have contract tests to validate API response against JSON Schema. I use [jsonschema](https://github.com/tdegrunt/jsonschema) package for that on my current project. It's a recommendation and you are not required to implement that.

**Review by maxm-dev — APPROVED** [review]
Good job!

File: product-service/src/functions/getProductById/__tests__/getProductById.test.ts (hunk 2 above)
Comment (alexanderteplov) [reply product-service/src/functions/getProductById/__tests__/getProductById.test.ts:38]: We use [ajv](https://ajv.js.org/) in our project, on the frontend. I think it should be connected with reusing Open API models. I try and consider this option someday.

File: product-service/serverless.ts (hunk 1 above)
Comment (alexanderteplov) [reply product-service/serverless.ts:40]: Read this from docs. Decided to skip it this time due to the small size of the section.

File: product-service/src/functions/getProductById/__tests__/getProductById.test.ts (hunk 2 above)
Comment (maxm-dev) [reply product-service/src/functions/getProductById/__tests__/getProductById.test.ts:38]: Yup, good choice. I had experience with ajv as well and it was quite positive.

**Pull request merged**